pennylane/qnn/torch.py

```diff
@@ -530,7 +530,7 @@ def input_arg(self):
         return self._input_arg
 
     @staticmethod
-    def set_input_argument(input_name: Text = "inputs") -> None:
+    def set_input_argument(input_name: str = "inputs") -> None:
```

Comment:
was something wrong with `Text`?

Reply:
lint was complaining that `typing.Text` is deprecated.

https://docs.python.org/3/library/typing.html#typing.Text